src/BusinessLogic/CheckoutAPI/PaymentMethods/Requests/GetCachedPaymentMethodsRequest.php

```diff
@@ -40,6 +58,38 @@ public function setMerchantId(string $merchantId): void
         $this->merchantId = $merchantId;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getShippingCountry(): ?string
+    {
+        return $this->shippingCountry;
+    }
```

Comment:
```suggestion
    /**
     * @return string
     */
    public function getShippingCountry(): string
    {
        return (string) $this->shippingCountry;
    }
```
related to https://github.com/sequra/integration-core/pull/38/files#r2399231555